client/header/index.js

```diff
@@ -33,8 +35,14 @@ export const Header = ( { sections, isEmbedded = false, query } ) => {
 	const rafHandle = useRef( null );
 	const threshold = useRef( null );
 	const siteTitle = getSetting( 'siteTitle', '' );
-	const _sections = Array.isArray( sections ) ? sections : [ sections ];
+	const _sections = useMemo(
```

Comment:
Eslint warns about this needing memoization to avoid unnecessary re-renders. so this fixes that.

Reply:
@psealock that seems reasonable. Will fix this.

Reply:
Fixed in https://github.com/woocommerce/woocommerce-admin/pull/5037/commits/c861ec08312226b8fe6db24ab6361e183bb82137